### ISSUE TITLE
[Steam UI][MusicPopup] Music popup(?) is glitching and unstable

### ISSUE THREAD

**Issue by deanrih** [ValveSoftware/steam-for-linux#9388]
#### Your system information

* Steam client version (build number or date): 1682634349
* Distribution (e.g. Ubuntu): Manjaro (Arch)
* Opted into Steam client beta?: [Yes/No] Yes
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:

The new Steam Client UI appears to add new Music popup on the top right corner after you play a music from a game, however if you click and open this popup, it doesn't stay still, here's the video recording:

[untitled.webm](https://user-images.githubusercontent.com/16235605/235111200-44c6ccb7-47e6-4015-ad1b-fe223ee79512.webm)

The provided recording however doesn't reflect the actual "unstableness", "IRL" it glitch much more frequently and you can even see two of each buttons.

#### Steps for reproducing this issue:

1. Open Steam Client UI
2. Open the menu View -> Soundtracks
3. Choose whicever locally installed soundtrack
4. Play the soundtrack
5. Stop
6. Open the Library (probably unecessary)
7. Click the Music popup on the top right corner (located beside the Family View and Announcement)
8. The popup view is glitching out

**Comment by deanrih** [comment]
The problem seems to be fixed, I don't actually know which version fixed it, but the version I used is 1683859267, also I don't know if I should close this issue or not even if the problem seems to be fixed, so I'll keep it open for now.

Thank you for the hard work!

**Comment by deanrih** [comment]
Update:

Trying the recently discovered Sneak Peek Beta branch of Steam, specifically version `1684288425` (Thank you it's finally selectable), and the PopUp seems broken again back to the before the fix.

**Comment by kisak-valve** [comment]
Hello @deanrih, are you still experiencing this issue on an up to date system?

**Comment by deanrih** [comment]
@kisak-valve Yes, it still happening, my Steam Client version is `1687386907` and the popout still moves around and makes it hard to play/pause. As for the OS itself, it is up to date although still using Kernel 6.2, haven't tried 6.3 yet since it has some problem with the NVIDIA driver

**Comment by deanrih** [comment]
Okay, I finally updated my kernel version to 6.3 (turned out I just need to lead the Kernel to use GCC instead of CC when building the kernel module), and yes, it is still happening.

**Comment by deanrih** [comment]
Tested on the beta branch version `1688171965`, the issue still persists.

**Comment by kjarosh** [comment]
I do have the same problem, here is the recording at 240 Hz (with my personal data censored):

[vibrating_soundtrack_controls.webm](https://github.com/ValveSoftware/steam-for-linux/assets/1507072/ed27cdd0-fc3e-48c0-b3c4-6fcf5e1a2071)

Example songs which cause this problem:

* Risk of Rain 2 — songs 1–8,10,12–14,18–21,
* Barotrauma — songs 1–14,17–18,20–21,23–25,27–29,30–33,35

I suspect it has something to do with the length of the text inside the popup. When the text is long enough that it needs to scroll, it works; but when the text is short, it doesn't. For instance, the soundtrack of Deep Rock Galactic has a long name and every song is displayed properly.

I have Fedora 38 fully up-to-date. My system info: https://gist.github.com/kjarosh/795a7610fd52465480f7b83f4decd325

**Comment by deanrih** [comment]
I know this is a very late update, but the popup so far still stable, closing this as completed.

**Comment by kjarosh** [comment]
It still does glitch out for me for some songs (e.g. the Risk of Rain 2 soundtrack).
